src/pk.c

```diff
@@ -11918,6 +11918,11 @@ int wolfSSL_EC_KEY_LoadDer_ex(WOLFSSL_EC_KEY* key, const unsigned char* derBuf,
         else {
             ret = wc_EccPublicKeyDecode(derBuf, &idx, (ecc_key*)key->internal,
                 derSz);
+            if (ret < 0) {
+                /* This could be a pair of points. */
```

Comment:
Please clarify this comment. If the public key decode fails, we try again as x.963 `[point type][x][opt y]`. Also make sure `idx` is reset in case it gets used...